Sources/Widgets/Widgets3D/ResliceCursorWidget/behavior.js

```diff
@@ -134,6 +145,7 @@ export default function widgetBehavior(publicAPI, model) {
       ScrollingMethods.MIDDLE_MOUSE_BUTTON
     ) {
       isScrolling = true;
+      publicAPI.resetUpdateMethod();
```

Comment:
why not also in `handleStartMouseWheel` ?

Reply:
Error.